mmdet3d/models/dense_heads/anchor3d_head.py

```diff
@@ -51,15 +51,15 @@ def __init__(self,
                      type='Anchor3DRangeGenerator',
                      range=[0, -39.68, -1.78, 69.12, 39.68, -1.78],
                      strides=[2],
-                     sizes=[[1.6, 3.9, 1.56]],
+                     sizes=[[3.9, 1.6, 1.56]],
                      rotations=[0, 1.57],
                      custom_values=[],
                      reshape_out=False),
                  assigner_per_size=False,
                  assign_per_class=False,
                  diff_rad_by_sin=True,
-                 dir_offset=0,
-                 dir_limit_offset=1,
+                 dir_offset=-np.pi / 2,
```

Comment:
Why is np.pi / 2 instead of np.pi / 4 (0.7854)?

Reply:
Because before refactor `dir_offset` is 0.